package.json

```diff
@@ -49,16 +49,17 @@
     "lint": "eslint --ext .js,.jsx ${npm_config_glob:-${npm_package_config_lint_glob}}",
     "lint:watch": "nodemon -e js,jsx --exec 'npm run lint --silent || true'",
     "postlint": "echo 'Lint Complete'",
-    "pretest": "echo 'Tests Starting:' && echo '  grep: '${npm_config_grep:-${npm_package_config_test_grep_unit} (default)} && echo '  glob: '${npm_config_glob:-${npm_package_config_test_glob} (default)}",
-    "test": "mocha --recursive --compilers js:babel-core/register ${npm_config_glob:-${npm_package_config_test_glob}}",
+    "pretest": "npm install -g npm-install-peers && npm-install-peers",
```

Comment:
This can't stay for a couple reasons:

1. It's evil to `npm install -g` from within a package's script
1. This runs every time you run `npm test`, which is terrible

Instead of creating a new problem by only having react in the peer dependencies and needing to install them somehow, I'm just going to add react and react-dom as `devDependencies` _as well as_ peerDependencies.  Then all of the `npm-install-peers` stuff can go away.